SPECS/kernel-hyperv/kernel-hyperv.spec

```diff
@@ -3,7 +3,7 @@
 Summary:        Linux Kernel optimized for Hyper-V
 Name:           kernel-hyperv
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
In general, it is not required to update the release number for `hyperv-daemons` or `kernel-hyperv` when updating `kernel.spec`.

Reply:
Reverted